.github/workflows/update.yml

```diff
@@ -16,7 +16,11 @@ jobs:
     strategy:
         fail-fast: false
         matrix:
+<<<<<<< HEAD
```

Comment:
I do not understand why you merged main.  (and develop) those merges have to  be done on the main repo of GSoC-pgRouting
by merging main you are making a mess.
You do not mix branches
Do a 
```
git fetch upstream
git rebase -i veenit-2021
```
where upstream is the gsoc repo
and remove any commit that is not your commit and remove the merges
and be very very very very careful, or you will mess up your branch
after the rebase
```
git push -f
```

Reply:
I'll fix it and merge it, its more difficult than expected